pybamm/util.py

```diff
@@ -343,8 +343,10 @@ def get_parameters_filepath(path):
 
 
 def have_jax():
-    """Check if jax is installed"""
-    return importlib.util.find_spec("jax") is not None
+    """Check if jax and jaxlib are installed"""
+    return (importlib.util.find_spec("jax") is not None) and (
+        importlib.util.find_spec("jaxlib") is not None
```

Comment:
Another approach would be to create a new `have_jaxlib` function to carry out this condition but I am not sure if that would be better

Reply:
Creating `have_jaxlib` would complicate things unnecessarily

Reply:
Yes